src/wrappers.rs

```diff
@@ -580,6 +625,10 @@ macro_rules! py_wrap_union_enum {
 
         $crate::impl_as_mut_for_wrapper!($name);
 
+        $(
+        $crate::py_wrap_union_enum!(@from $name, $rs_inner, $variant_name, $variant $($(=> $convert)+)?);
+        )+
+
         $crate::paste::paste! {
         #[$crate::pyo3::pymethods]
         impl $name {
```

Comment:
Not a blocker, but how does `#[new]` (line 636) work with fieldless variants?

Also, I'm having trouble running the tests locally with `cargo make test`:

```
dyld[38322]: Library not loaded: @rpath/libpython3.9.dylib
```

Reply:
I had to do a `cargo clean` in order to get tests to compile. I think something gets cached that shouldn't be, but I don't know what.

Reply:
> Not a blocker, but how does #[new] (line 636) work with fieldless variants?

It doesn't :/ The `#[new]` only works for variants with associated data, which is why `new_unit` needed to be added.